pages/linux/enum4linux.md

```diff
@@ -0,0 +1,24 @@
+# enum4linux
+
+> Tool for enumerating Windows and Samba information from remote systems.
+> It attempts to offer similar functionality to enum.exe formerly available from www.bindview.com.
```

Comment:
Should we have `enum.exe` inside backticks here? I've seen multiple forms of writing it here on tldr.

Reply:
Probably yes. The problem is that this has never been standardized, i.e. codified into the style guide.